libero/src/main/groovy/com/connexta/libero/Libero.groovy

```diff
@@ -1,22 +1,29 @@
 package com.connexta.libero
 
+import org.ajoberstar.grgit.Grgit
+import org.apache.maven.shared.invoker.DefaultInvocationRequest
+import org.apache.maven.shared.invoker.DefaultInvoker
+import org.apache.maven.shared.invoker.InvocationRequest
+import org.apache.maven.shared.invoker.Invoker
+
 import java.text.SimpleDateFormat
 
 // TODO: What should be the process when the build fails? How should the fixes be handled?
 //       should that be part of the script? Should the script have an option to only do a build?
-// TODO: Add check for existing tag
 // TODO: Add option to clone the repository
 // TODO: Check if remote is a url or just a name
 // TODO: add option to clean artifacts from local .m2
-// TODO: add user prompts to get better information when user provides bad input (maybe)
 // TODO: add verification step for git ref
 //  see git rev-parse for validation
-// TODO: change currentVersion finder to use git show rev:file to get the xml content
 
 class Libero {
 
     Config config
     Util util
+    Grgit git
```

Comment:
We should keep this class stateless, otherwise the Libera script will have to create a new instance every time it runs it.